com.unity.render-pipelines.high-definition/Documentation~/HDRP-Asset.md

```diff
@@ -84,7 +84,7 @@ These settings control the draw distance and resolution of the decals atlas that
 | **-- Use Optimal Settings**     | Enable the checkbox to make DLSS control the Sharpness and Screen Percentage automatically. |
 | **-- Sharpness**                | Controls how the DLSS upsampler renders edges on the image. More sharpness usually means more contrast and a clearer image but can increase flickering and fireflies. Unity ignores this property if you enable **Use Optimal Settings**. |
 | **- Dynamic Resolution Type**   | Use the drop-down to select the type of dynamic resolution HDRP uses:<br />&#8226; **Software**: This option allocates render targets to accommodate the maximum resolution possible, then rescales the viewport accordingly. This allows the viewport to render at varying resolutions. |
-| **- Upscale Filter**            | Use the drop-down to select the filter that HDRP uses for upscaling.<br />&#8226; **Bilinear**: A low quality upsample. The least resource intensive option.<br />&#8226; **Catmull-Rom**: A bicubic upsample with 4 taps.<br />&#8226; **Lanczos**: A sharp upsample. This method can potentially introduce artifacts so you should not use it for extreme upsampling cases for example, when the screen percentage is less than 50%.<br />&#8226; **Contrast Adaptive Sharpen**: An ultra sharp upsample. This option is not meant for screen percentages less than 50% and still sharpens when the screen percentage is set to 100%. This uses **FidelityFX (CAS) AMD™**. For information about FidelityFX and Contrast Adaptive Sharpening, see [AMD FidelityFX](https://www.amd.com/en/technologies/radeon-software-fidelityfx). |
+| **- Upscale Filter**            | Use the drop-down to select the filter that HDRP uses for upscaling.<br />&#8226; **Bilinear**: A low quality upsample. The least resource intensive option.<br />&#8226; **Catmull-Rom**: A bicubic upsample with 4 taps.<br />&#8226; **Lanczos**: A sharp upsample. This method can potentially introduce artifacts so you should not use it for extreme upsampling cases for example, when the screen percentage is less than 50%.<br />&#8226; **Contrast Adaptive Sharpen**: An ultra sharp upsample. This option is not meant for screen percentages less than 50% and still sharpens when the screen percentage is set to 100%. This uses **FidelityFX (CAS) AMD™**: For information about FidelityFX and Contrast Adaptive Sharpening, see [AMD FidelityFX](https://www.amd.com/en/technologies/radeon-software-fidelityfx). <br />&#8226; **FidelityFX Super Resolution 1.0 AMD™**: a spatial super-resolution technology that leverages cutting-edge algorithms to produce impressive upscaling quality at very fast performance.|
```

Comment:
Unrelated to this change, but is this even true for CAS

> An ultra sharp upsample. This option is not meant for screen percentages less than 50% and still sharpens when the screen percentage is set to 100%

Unless something got changed recently, CAS is part of Dynamic res and that is turned off for anything but DLSS when resolution is 100%

Reply:
Also, if we want to make it work at 100%, we should have the option elsewhere. (To do in a different PR, as I said, unrelated to this)

Reply:
What do you mean? take RCAS and FSR completely out of DRS panel?

Reply:
Nope, have the "run with 100% screen percentage" option outside (so CAS outside of DRS). Not sure if is better to force it running when 100% too as well, but that sounds somewhat limiting (thinking of like temporal upscale, with that you might still want to run CAS after, but being both filters of upscale it won't be possible).

Reply:
gotit. I think it shouldnt be an option, should be on %100 of the time dont you think?

Reply:
Just for these passes*

Reply:
Well yes, but still we want to have an option somewhere outside for when DRS  is _not_ used. 

If I run normal rendering, it is not an obvious UX experience to enable DRS if I never intend to do DRS, just to get a sharpening filter on my normal 100% rendering.